Tests/src/artofillusion/object/SceneCameraTest.java

```diff
@@ -0,0 +1,504 @@
+/* Copyright (C) 2020 by Maksim Khramov
+
+   This program is free software; you can redistribute it and/or modify it under the
+   terms of the GNU General Public License as published by the Free Software
+   Foundation; either version 2 of the License, or (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful, but WITHOUT ANY 
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A 
+   PARTICULAR PURPOSE.  See the GNU General Public License for more details. */
+
+
+package artofillusion.object;
+
+import artofillusion.Camera;
+import artofillusion.Scene;
+import artofillusion.animation.FilterParameterTrack;
+import artofillusion.animation.PoseTrack;
+import artofillusion.animation.WeightTrack;
+import artofillusion.image.ComplexImage;
+import artofillusion.image.filter.ImageFilter;
+import artofillusion.math.BoundingBox;
+import artofillusion.math.CoordinateSystem;
+import artofillusion.test.util.StreamUtil;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InvalidObjectException;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ *
+ * @author MaksK
+ */
+public class SceneCameraTest
+{
+  
+  @Test
+  public void testGetCameraBounds() {
```

Comment:
Is there anything that would break if we changed the size or geometry of the visual mesh that is used to represent a `SceneCamera`? If not, I'd submit that we have no need to run such a test, and the test will just make it harder to change the visuals in the future if we need to.